### PR TITLE
NHC bits

### PR DESCRIPTION
**Pull request by heitorPB**
this pr depends on https://github.com/omnivector-solutions/slurm-ops-manager/pull/44

### CODE REVIEW

File: charm-slurmd/src/charm.py [hunk 1]
```diff
@@ -108,6 +115,9 @@ def _on_check_status_and_write_config(self, event):
         # Ensure we aren't dealing with a StoredDict before trying
         # to render the slurm.conf.
         slurm_config = dict(slurm_config)
+        # NOTE: slurm_config_nhc_values allows to change the interval and note
+        # state. Could be turned into metadata.yaml options later.
```
Comment (jamesbeedy) [inline charm-slurmd/src/charm.py]: `metadata.yaml` -> `config.yaml`

File: tests/20-nhc-config.bats [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+@test "Assert we can update nhc.conf" {
+	nhc_conf="/etc/nhc/nhc.conf"
+
+	juju config slurmd nhc-conf="# new config"
+	sleep 5 # just to be sure all units updated correctly
+
+	run juju run --unit slurmd/leader "tail -n 4 $nhc_conf"
+	assert_output --partial "# new config"
+}
+
```
Comment (jamesbeedy) [inline tests/20-nhc-config.bats]: remove newline

File: charm-slurmd/src/charm.py (hunk 1 above)
Comment (heitorPB) [reply charm-slurmd/src/charm.py]: ouch, thanks for noticing it!

**Review by jamesbeedy — APPROVED** [review]
👍

**Pull request merged**